src/main/java/seedu/address/logic/parser/meeting/ParserUtil.java

```diff
@@ -52,13 +54,15 @@ public static Title parseTitle(String title) throws ParseException {
      *
      * @throws ParseException if the given {@code phone} is invalid.
      */
-    public static Description parseDescription(String description) throws ParseException {
-        requireNonNull(description);
-        String trimmedDescription = description.trim();
-        if (!Description.isValidDescription(trimmedDescription)) {
+    public static OptionalDescription parseDescription(Optional<String> description) throws ParseException {
```

Comment:
Generalise this so that deliverables and contacts can also use the 'parseDescription'